client/extensions/woocommerce/store-sidebar/index.js

```diff
@@ -145,6 +145,28 @@ class StoreSidebar extends Component {
 		);
 	}
 
+	promotions = () => {
```

Comment:
This needs a `config.isEnabled` check, otherwise the link will show in production and other environments.

Reply:
Good catch, thanks! Added the check.